### PR TITLE
Force remove test repo root path in case previous test is still locking it

### PR DESCRIPTION
**Pull request by lafriks**
Wait up to 2 seconds to remove previous test repository root directory to make tests more stable

### CODE REVIEW

**Review by appleboy — APPROVED** [review]

File: models/unit_tests.go [hunk 1]
```diff
@@ -72,6 +73,18 @@ func createTestEngine(fixturesDir string) error {
 	return InitFixtures(&testfixtures.SQLite{}, fixturesDir)
 }
 
+func forcedRemoveAll(dir string) error {
```
Comment (Bwko) [inline models/unit_tests.go]: Could you rename this method? It's not really forcing anything

Comment (lafriks) [reply models/unit_tests.go]: Any suggestions?

Comment (Bwko) [reply models/unit_tests.go]: Maybe something like this?:

```
func retryingRemoveAll(dir string, retries int) error {
```

Comment (lafriks) [reply models/unit_tests.go]: @Bwko done

**Review by Bwko — APPROVED** [review]

**Pull request merged**